src/guest_memory.rs

```diff
@@ -302,14 +303,14 @@ pub trait GuestMemoryRegion: Bytes<MemoryRegionAddress, E = Error> {
 /// implementation, providing a zero-cost abstraction whenever immutable memory
 /// maps are sufficient.
 ///
+/// # Examples (uses the `backend-mmap` and `backend-atomic` features)
+///
 /// ```
+/// ##[cfg(feature = "backend-mmap")]
```

Comment:
Ahh, this is probably because I have to remove all the `#[cfg(...)]`s to get fmt to actually work on the doc comments. Must have made some error when I unremoved them.